src/shogun/regression/LeastAngleRegression.h

```diff
@@ -183,13 +205,385 @@ class CLeastAngleRegression: public CLinearMachine
 	/** @return object name */
 	virtual const char* get_name() const { return "LeastAngleRegression"; }
 
+
+	template <typename ST>
+	static vector<ST> make_vector(int32_t size, ST val)
+	{
```

Comment:
Why would we want to increment the values of a vector (as the entries change)?  Whoever implemented this is only using it to make vectors with all 0 entries, so I'll just replace those lines with a memset to zero.